curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-fetch-method.md

```diff
@@ -38,6 +38,31 @@ Update the code to create and send a `GET` request to the freeCodeCamp Cat Photo
 
 # --hints--
 
+
+Api call yeilds expected results
```

Comment:
This isn't the typical syntax we take for hint messages.

Ideally, we follow a "Your X should Y" format.

Reply:
I would think the most appropriate name would be, "Your code should get the element with id message and change its inner HTML to the string of JSON data." although there is already a test with that message that isnt doing what its supposed to, so in this case kindly specify how you would like me to name it and ill do it.

Reply:
Since the original test isn't covering everything we could use

> Your code should get the element with id `message` and replace its inner HTML

for that and

> Your code should use the fetched data to replace the inner HTML

for this test.

Reply:
yes ive done that